fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -227,16 +227,23 @@ def execute(self, context):
 			isCustomImport = context.scene.ootDLImportUseCustomPath
 			scale = context.scene.ootActorBlenderScale
 			basePath = bpy.path.abspath(context.scene.ootDecompPath)
+			importAll = context.scene.ootDLImportAll
 			removeDoubles = context.scene.ootDLRemoveDoubles
 			importNormals = context.scene.ootDLImportNormals
 			drawLayer = bpy.context.scene.ootDLImportDrawLayer
 
 			filepaths = [ootGetObjectPath(isCustomImport, importPath, folderName)]
 			if not isCustomImport:
+				if importAll:
+					raise PluginError("Import All DLs requires custom import path!")
```

Comment:
I can't make sense of `ootGetObjectPath` 😅 why does importAll need isCustomImport

Reply:
A *non*-custom import means fast64 will look for the DL name in `gameplay_keep` (and possibly `gameplay_field_keep` or `gameplay_dangeon_keep`) as well as in object files. So first of all, using non-custom import for scene import won't work because the scene DLs aren't in the object files. And second, import all with a non-custom import will import every DL in `gameplay_keep` (and possibly more)!

Reply:
I may be a bit lost about the intent, does this PR only allow importing all dlists from a scene or all dlists from any .c file?

Reply:
What it actually does, assuming custom import, is all DLs from any C file. This is probably only useful for scenes (meaning scenes + rooms).